### PR TITLE
Use Butter Knife

### PR DESCRIPTION
**Pull request by rykann**

### CODE REVIEW

File: app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java [hunk 1]
```diff
@@ -33,42 +34,38 @@
...
 
-                        @Override
-                        public void onFailure(Throwable t) {
-                            Log.e(LOG_TAG, "Login information invalid");
-                            Toast.makeText(LoginActivity.this, getString(R.string.email_password_invalid),
-                                    Toast.LENGTH_SHORT).show();
-                        }
-                    });
+    @OnClick(R.id.btnLogin)
+    void logIn() {
```
Comment (wangwh03) [inline app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java:43]: private void?

**Comment by wangwh03** [comment]
Looks really clean, shipit!

File: app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java (hunk 1 above)
Comment (rykann) [inline app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java:43]: I tried that, but Butter Knife complained at compile time: `@OnClick methods must not be private or static.`

Comment (wangwh03) [inline app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java:43]: Ah I see, thanks for letting me know!

**Pull request merged**